src/chat/Chat.js

```diff
@@ -78,6 +77,9 @@ export default class Chat extends PureComponent<Props> {
                 />
               </ActionSheetProvider>
             )}
+            <View style={styles.floatingView}>
+              <UnreadNoticeContainer />
```

Comment:
Can you just apply a `styles` object to `UnreadNoticeContainer`?

Reply:
Done